test/fixtures/simple_example/terraform.tfvars

```diff
@@ -0,0 +1,3 @@
+project_id = "<project_id>"
```

Comment:
`.tfvars` files should not be included as they are environment dependent and changes should not be tracked.

Reply:
Resolved in
https://github.com/terraform-google-modules/terraform-google-dataflow/pull/1/commits/17a9d4f730a5f4794101ea7bd1f8baf231f8fee1